libcudacxx/test/libcudacxx/libcxx/macros/prefetch.compile.pass.cpp

```diff
@@ -0,0 +1,20 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#include <cuda/std/detail/__config>
+
+#include <test_macros.h>
+
+int main(int, char**)
+{
+  int memory[8];
+  _CCCL_BUILTIN_PREFETCH(memory, 0, 1);
```

Comment:
the builtin takes a variadic number of arguments, so we should make sure that all 3 versions of it pass